tests/sentry/incidents/test_action_handlers.py

```diff
@@ -135,7 +135,7 @@ def test_user(self):
             handler.fire()
         out = mail.outbox[0]
         assert out.to == [self.user.email]
-        assert out.subject.startswith("Alert Rule Fired")
+        assert out.subject == u"[Open] {} - {}".format(incident.title, self.project.slug)
```

Comment:
This may be related to the resolved subject, but we would expect this to be either warning or critical I think

Reply:
Yeah I'd expect this to be warning/critical. The problem is that the incident is being created via a path that doesn't happen in production now, so you'll need to manually set the status here as well.